backend/src/models/Image.js

```diff
@@ -0,0 +1,7 @@
+export default {
+  url: { primary: true, type: 'string', uri: { allowRelative: true } },
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-02-20T19:43:24Z" title="Thursday, February 20th 2020, 8:43:24 pm +01:00">Feb 20, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,7 @@
+export default {
+  url: { primary: true, type: 'string', uri: { allowRelative: true } },
```


```suggestion
  url: { primary: true, type: 'string', unique: 'true', uri: { allowRelative: true } },
```

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-02-20T19:44:42Z" title="Thursday, February 20th 2020, 8:44:42 pm +01:00">Feb 20, 2020</time>_

---


this in my mind would make the extra `migration` unnecessary, oder?

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-02-21T21:40:46Z" title="Friday, February 21st 2020, 10:40:46 pm +01:00">Feb 21, 2020</time>_

---


Primary keys are always unique. The extra migration is an training for us to create migrations when we setup new indices. So, during a helm upgrade the migrations will run but no `init`.

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-02-25T14:34:55Z" title="Tuesday, February 25th 2020, 3:34:55 pm +01:00">Feb 25, 2020</time>_

---


great, thanks for the explanation